### DISCUSSION TITLE
[A-L] (2023/24) Foglio 4 - Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#185] [Esercizi - Viaggi/Piazza]
![Screenshot 2023-10-28 110409](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/accd98cf-12cb-49c7-ab31-fedba52e0200)

**Comment by IonutCicio** [comment]
_(Nota: ho usato la notazione_ $\equiv _n$ _per indicare la congruenza modulo_ $n$ _)._ 
È noto che la soluzione del sistema esiste, è unica ed è nella forma

$$
\tilde{x} = R_1\tilde{x}_1 + R_2\tilde{x}_2 + R_3\tilde{x}_3
$$

Dove, dato $R = 5 \cdot 7 \cdot 11 = 385$, si calcolano

$$R_1 = \frac{R}{5} = 7 \cdot 11 = 77$$

$$R_2 = \frac{R}{7} = 5 \cdot 11 = 55$$

$$R_3 = \frac{R}{11} = 5 \cdot 7 = 35$$

Ora si possono trovare $\tilde{x} _1, \tilde{x} _2, \tilde{x} _3$

$$R_1 \tilde{x}_1 \equiv _5 3 \implies 77 \tilde{x}_1 \equiv _5 3 \implies 2 \tilde{x}_1 \equiv _{5} 3 \implies \tilde{x}_1 \equiv _5 4 $$

$$R_2 \tilde{x}_2 \equiv _7 4 \implies 55 \tilde{x}_2 \equiv _7 4 \implies 6 \tilde{x}_2 \equiv _7 4 \implies \tilde{x}_2 \equiv _7 3$$

$$R_3 \tilde{x}_3 \equiv _{11} 4 \implies 35 \tilde{x}_3 \equiv _{11} 4 \implies 2 \tilde{x}_3 \equiv _{11} 4 \implies \tilde{x}_3 \equiv _{11} 2 $$

Quindi $\tilde{x} = 77 \cdot 4 + 55 \cdot 3 + 35 \cdot 2 \equiv _{385} 543 \equiv _{385} 158$. Infatti $158 \equiv _5 3, 158 \equiv _7 4, 158 \equiv _{11} 4$

**Comment by Elia-Belli** [comment]
Soluzione analoga a @CuriousCI 
Faccio solo notare che la soluzione esiste ed è unica perchè il sistema è cinese e $(5,7)=1,(5,11)=1,(7,11)=1$, quindi vale il Teorema Cinese del Resto
